MoEngagePluginBase/MoEngagePluginUtils.swift

```diff
@@ -27,6 +27,21 @@ public class MoEngagePluginUtils {
         return appIdDict
     }
     
+    static public func fetchShouldTrackUserAttrBooleanAsNumberFromPayload(attribute: [String: Any]) -> Bool {
```

Comment:
Verify these methods if the flag should be instance specific.

Reply:
Addressed.